notes/interview.vbs

```diff
@@ -7756,24 +7927,53 @@ function app_month_utility_detail(determined_utilities, heat_expense, ac_expense
 	If phone_expense = True then phone_checkbox = checked
 	If none_expense = True then none_checkbox = checked
 
+	dlg_len = 175
+
+	interview_note_details_exists = False
+	intvw_notes_len = 20
+
+	If trim(question_15_interview_notes) <> "" Then
+		interview_note_details_exists = True
+		intvw_notes_len = intvw_notes_len + 40
+	End If
+
+	If interview_note_details_exists = True Then dlg_len = dlg_len + intvw_notes_len + 10
+
 	Do
 		current_utilities = all_utilities
 
 		Dialog1 = ""
-		BeginDialog Dialog1, 0, 0, 246, 175, "Determination of Utilities in Month of Application"
-		  CheckBox 30, 45, 50, 10, "Heat", heat_checkbox
-		  CheckBox 30, 60, 65, 10, "Air Conditioning", ac_checkbox
-		  CheckBox 30, 75, 50, 10, "Electric", electric_checkbox
-		  CheckBox 30, 90, 50, 10, "Phone", phone_checkbox
-		  CheckBox 30, 105, 50, 10, "NONE", none_checkbox
-		  ButtonGroup ButtonPressed
-		    PushButton 170, 105, 65, 15, "Calculate", calculate_btn
-		    PushButton 170, 155, 65, 15, "Return", return_btn
-		  Text 10, 10, 235, 10, "Check the boxes for each utility the household is responsible to pay:"
-		  GroupBox 15, 30, 225, 95, "Utilities"
-		  Text 150, 45, 50, 10, "$ " & determined_utilities
-		  Text 150, 60, 35, 35, all_utilities
-		  Text 15, 135, 225, 20, "Remember, this expense could be shared, they are still considered responsible to pay and we count the WHOLE standard."
+		BeginDialog Dialog1, 0, 0, 246, dlg_len, "Determination of Utilities in Month of Application"
+			CheckBox 30, 45, 50, 10, "Heat", heat_checkbox
+			CheckBox 30, 60, 65, 10, "Air Conditioning", ac_checkbox
+			CheckBox 30, 75, 50, 10, "Electric", electric_checkbox
+			CheckBox 30, 90, 50, 10, "Phone", phone_checkbox
+			CheckBox 30, 105, 50, 10, "NONE", none_checkbox
+
+			Text 10, 10, 235, 10, "Check the boxes for each utility the household is responsible to pay:"
+			GroupBox 15, 30, 225, 95, "Utilities"
+			Text 150, 45, 50, 10, "$ " & determined_utilities
+			Text 150, 60, 35, 35, all_utilities
+			Text 15, 135, 225, 20, "Remember, this expense could be shared, they are still considered responsible to pay and we count the WHOLE standard."
+
+			y_pos = 160
+			'displaying details from interview notes in the dialog for calculating app month utilities
+			If interview_note_details_exists = True Then
+				GroupBox 5, y_pos, 235, intvw_notes_len, "Interview NOTES entered into the Script already"
+				y_pos = y_pos + 15
+				If trim(question_15_interview_notes) <> "" Then
+					Text 10, y_pos, 215, 10, "15. Does your household have utility expenses any time during the year?"
+					Text 15, y_pos+10, 210, 30, question_15_interview_notes
+					y_pos = y_pos + 40
+				End If
+				y_pos = y_pos + 10
+			Else
+				y_pos = y_pos - 5
+			End If
+
+			ButtonGroup ButtonPressed
+				PushButton 170, 105, 65, 15, "Calculate", calculate_btn
+				PushButton 170, y_pos, 65, 15, "Return", return_btn
 		EndDialog
 
 		dialog Dialog1
```

Comment:
Line 8061 Spelling Error: Denials can be coded in REPT/PND2...Otherise, since the interview should be done at this point" should be "Otherwise"

Reply:
Spelling fixed.